src/test/java/marquez/db/UtilsTest.java

```diff
@@ -0,0 +1,83 @@
+package marquez.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.sql.Array;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import marquez.UnitTests;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(UnitTests.class)
+public class UtilsTest {
+  @Test
+  public void testToInstantOrNull() {
+    final Instant expected = Instant.now();
+    final Timestamp timestamp = Timestamp.from(expected);
+    final Instant actual = Utils.toInstantOrNull(timestamp);
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testToInstantOrNull_nullTimestamp() {
+    final Timestamp nullTimestamp = null;
+    final Instant instant = Utils.toInstantOrNull(nullTimestamp);
+    assertNull(instant);
+  }
+
+  @Test
+  public void testToUuidOrNull() {
+    final String uuidString = "4f4c72c1-281a-4b82-a4e0-9c909a97832f";
+    final UUID expected = UUID.fromString(uuidString);
+    final UUID actual = Utils.toUuidOrNull(uuidString);
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testToUuidOrNull_nullUuidString() {
+    final String nullUuidString = null;
+    final UUID uuid = Utils.toUuidOrNull(nullUuidString);
+    assertNull(uuid);
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testToUuidOrNull_throwsException_onEmptyUuidString() {
+    final String emptyUuidString = "";
+    Utils.toUuidOrNull(emptyUuidString);
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testToUuidOrNull_throwsException_onBlankUuidString() {
+    final String blankUuidString = " ";
+    Utils.toUuidOrNull(blankUuidString);
+  }
+
+  @Test
+  public void testToListOrEmpty() throws SQLException {
```

Comment:
I don't think this function name matches the test (at least the empty list part). It's important to add a test for an empty list, and I don't think I see that case exercised. Can we add it?